addons/binding/org.openhab.binding.meterreader/src/main/java/gnu/io/factory/RxTxPortCreator.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
```

Comment:
I assume most of this gnu.io code here is NOT under EPL and should actually not be part of this binding - you are still waiting for https://github.com/NeuronRobotics/nrjavaserial/pull/109 and a new nrjavaserial release to get rid off this here, is this correct?

Reply:
`gnu.io.factory` package was created by myself. I tried to integrate them into nrjavaserial, but I guess it will never happen. Purpose is to let the factory decide about the implementation depending on the URL. In my case (and also few others are using it) I need a TelnetSerialPort (RFC2217).
Should I leave them only in this bundle and rename the package?

The other bundle `gnu.io.rfc2217` actually belongs to nrjavaserial. I'm waiting as you said for https://github.com/NeuronRobotics/nrjavaserial/pull/109 to be merged and new version integrated in openhab.

Reply:
Since I use RFC2217 the reconnection is faster and more reliable if I reboot my meter raspi.
I won't miss RFC2217. I would like to have more bindings with RFC2217 because my wiznet TCP modbus gateways should have RFC2217 too.

Reply:
> Should I leave them only in this bundle and rename the package?

Functionally, it does not belong in here at all, so I would prefer to see it addressed in nrjavaserial. If this takes too long, I'd be fine with further maintaining our fork https://github.com/openhab/nrjavaserial meanwhile and add it in there.

If we decide to factor the IEC stuff out, would that remove this requirement here as well? Or is rfc2217 also needed for the SML support?

Reply:
Ok, then I'll create a PR in that project, ok?

Reply:
I've created a new PR here:
https://github.com/openhab/nrjavaserial/pull/3